docs/apache-airflow-providers-sqlite/connections/sqlite.rst

```diff
@@ -23,14 +23,22 @@ The SQLite connection type provides connection to a SQLite database.
 
 Configuring the Connection
 --------------------------
-Host (required)
-    The host to connect to.
+Host (optional)
+    The host to connect to. This can either be a file on disk or an in-memory database. If not set, an in-memory database is being used.
 
-Schema (optional)
-    Specify the schema name to be used in the database.
+Extra (optional)
+    Specify the extra parameters (as json dictionary) that can be used in the sqlite connection.
+    See `Recognized Query Parameters <https://www.sqlite.org/uri.html>`_ for all supported parameters.
 
-Login (required)
-    Specify the user name to connect.
+URI format example
+^^^^^^^^^^^^^^^^^^
 
-Password (required)
-    Specify the password to connect.
+If serializing with Airflow URI:
+
+.. code-block:: bash
+
+   export AIRFLOW_CONN_SQLITE_DEFAULT='sqlite://host/?mode=ro'
```

Comment:
Host doesn't make any sense for a sqlite example

Reply:
This is the path to the database. In this example the file "host" will be created in the current directory.

Reply:
What do you want me to put there instead?

Reply:
Oh, have it as something like `sqlite:///path/to/db` then. And if it should only be `//` not `///` I think it's worth calling that out explicitly (as anyone familiar with sqlite might instinctively go for `///`)

Reply:
That said... Why are we changing it away from sqlite norms? Why can't we have `AIRFLOW_CONN_SQLITE_DEFAULT=sqlite:///path/to/db`? (Sorry, I've lost any and all context to this PR of late)

Reply:
🤔 good question actually.. I will check tomorrow if we can remove the `replace` in `get_uri`.

Reply:
I think it is because of `get_uri` impl in connection. It always has two slashes. So we need to add another one. Even if we add an absolute path in the host field we are still missing one. The absolute path in sql alchemy requires four slashes in total.

tldr; The URI you see in the docs is a connection URI - not a sqlalchemy uri.

Reply:
Thinking about it -- how does your PR behave if you try to specify a path to an absolute location? How would you create a URI for a db in `/tmp/my.sqlite`?

Reply:
`sqlite:///tmp/my.sqlite`

Reply:
You basically set the host field to the path and it will add another slash when getting the uri used for sql alchemy uris. For SQLite uris it will use `file:` instead.

Reply:
Since this is quite confusing, I will add an example with a relative path and one with an absolute path.